src/dist.rs

```diff
@@ -315,13 +337,77 @@ impl Dist {
     }
 }
 
+#[cfg(feature = "sass")]
+fn sass(
+    static_dir: &std::path::Path,
+    dist_dir: &std::path::Path,
+    options: sass_rs::Options
+) -> Result<()> {
+    fn is_sass(path: &std::path::Path) -> bool {
+        matches!(
+            path.extension().map(|x| x.to_str()).flatten(),
+            Some("sass") | Some("scss")
+        )
+    }
+
+    fn should_ignore(path: &std::path::Path) -> bool {
+        path
+            .file_name()
+            .expect("static files contains non-UTF8")
+            .to_str()
+            .map(|x| x.starts_with("_"))
+            .unwrap_or(false)
+    }
+
+    log::trace!("Generating dist artifacts");
+    let walker = walkdir::WalkDir::new(&static_dir).into_iter();
+    for entry in walker
+        .filter_map(|x| match x {
+            Ok(x) => Some(x),
+            Err(err) => {
+                log::error!("could not walk into directory `{}`: {}", &static_dir.display(), err);
```

Comment:
This should probably fail the process instead of being some kind of warning and continuing the process. I doubt the user would want a half-baked dist dir